2.4/contrib/httpd_util.sh

```diff
@@ -0,0 +1,14 @@
+# source
+
+export HTTPD_SERVER_ADMIN=${HTTPD_SERVER_ADMIN:-default@localhost}
+
+function configure_httpd() {
+  echo "---> Configuring Apache"
+  for template in $(find /opt/openshift/httpd/ -name '*.template'); do
```

Comment:
the /opt/openshift dir is unfortunate and will change (RHCSL wants to occupy the /usr/local/etc)